template/cmd_main.tmpl

```diff
@@ -7,10 +7,17 @@ import (
 	"{{ .RelDir }}"
 	"{{ .RelDir }}/server"
 	"{{ .RelDir }}/{{ .SnakeCaseName }}/cmd"
+	"github.com/mattn/go-colorable"
+	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 )
 
 func main() {
+	logrus.SetOutput(colorable.NewColorableStdout())
+	formatter := new(logrus.TextFormatter)
+	formatter.ForceColors = true
```

Comment:
If you don't like this forcibly set true, I'll fix to use isatty.IsTerminal here.

Reply:
I get colors by default? On servers we don't want that though probably, what's the reason for forcing color?

Reply:
logrus write escape sequence on command promplt event command prompt does not support escape sequence.